server.js

```diff
@@ -1,7 +1,16 @@
-var express = require('express');
-var app = express();
-app.get('/ping', (req, res) =>{
-    res.send('hello ')
-})
-
-var server = app.listen(3000);
+
+var express = require('express');

```

Comment:
Remove the duplicated server code. It's not clear why the same code is repeated twice. [important]